ui/components/ui/box/box.stories.js

```diff
@@ -220,7 +222,11 @@ export const DefaultStory = (args) => {
       />,
     );
   }
-  return <Box {...rest}>{children}</Box>;
+  return (
+    <Box {...rest} borderColor={COLORS.BORDER_MUTED}>
+      {children}
+    </Box>
+  );
 };
 
 DefaultStory.storyName = 'Default';
```

Comment:
Box story still works the same as before

### Before

<img width="1440" alt="Screenshot 2022-12-11 at 6 58 54 PM" src="https://user-images.githubusercontent.com/8112138/206958873-b55ee1b7-ef43-4019-a619-c85d6f6b42e8.png">


### After

<img width="1440" alt="Screenshot 2022-12-11 at 6 58 11 PM" src="https://user-images.githubusercontent.com/8112138/206958888-d2aadad2-de2e-46ab-93b1-7f5ec8acd59d.png">